src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Commands/ResumeAzureRmAnalysisServicesServer.cs

```diff
@@ -0,0 +1,60 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.AnalysisServices.Models;
+using Microsoft.Azure.Management.Analysis;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.AnalysisServices.Properties;
+using Microsoft.Azure.Management.Analysis.Models;
+
+namespace Microsoft.Azure.Commands.AnalysisServices
+{
+    [Cmdlet(VerbsLifecycle.Resume, "AzureRmAnalysisServicesServer", 
+        SupportsShouldProcess = true),
+     OutputType(typeof(List<AnalysisServicesServer>))]
+    [Alias("Resume-AzureAs")]
+    public class ResumeAzureAnalysisServicesServer : AnalysisServicesCmdletBase
+    {
+        [Parameter(Position = 1, ValueFromPipelineByPropertyName = true,
+            Mandatory = false, HelpMessage = "Name of resource group under which to retrieve the server.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Position = 0, ValueFromPipelineByPropertyName = true,
+            Mandatory = true, HelpMessage = "Name of a specific server.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Do not ask for confirmation.")]
+        public SwitchParameter Force { get; set; }
```

Comment:
Why use a Force parameter?  Force shoudl only be used if you have special conditions that might require a prompt.  If there is no check for special conditions, there is no need for a Force parameter